README.md

```diff
@@ -25,6 +33,32 @@
   - [Releasing seems weird?](#releasing-seems-weird)
 
 ---
+## Installation
+
+```bash
+yarn install
+```
+
+## Running and the application
+
+TODO write about and link to immutable webapps
+
+### Environment Variables
+
+| Name | Description | Example |
+| ---  | ----------- | ------- |
+| API_URL | The fully qualified URL for the endpoint gateway | http://localhost:5000 |
+| PUBLIC_URL | The fully qualified URL for the server hosting the assets | http://localhost:4000 |
+| HOST | The host used for the webpack dev server in local development | localhost |
+| PORT | The port used for the webpack dev server in local development | 9000 |
+
+### Running the application locally
+
+//
```

Comment:
make sure to say to make the public_url and host/port same in local when running with dev server